src/appleseed.shaders/src/blenderseed/as_closure2surface.osl

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2018 Jon Dent
```

Comment:
This should be

    // Copyright (c) 2018 Jon Dent, The appleseedhq Organization